human_robot_collaboration/package.xml

```diff
@@ -31,12 +31,17 @@
   <buildtool_depend>catkin</buildtool_depend>
   <build_depend>roscpp</build_depend>
 
+  <build_depend>ros_speech2text</build_depend>
+ 
+  <exec_depend>ros_speech2text</exec_depend>
+
```

Comment:
instad of adding a `build` and an `exec` depend, just add a `depend` (see lines below)